src/applications/model/three-lognormal-random-variable.h

```diff
@@ -82,14 +82,14 @@ class ThreeLogNormalRandomVariable : public LogNormalRandomVariable
    * \brief Returns a random double from a three-parameters log-normal distribution with the current mu and sigma.
    * \return A floating point random value.
    */
-  virtual double GetValue (void);
+  virtual double GetValue (void) override;
```

Comment:
not overriding the destructor..

Reply:
Are you referring to `virtual ~ThreeLogNormalRandomVariable ()`?
I fixed the destructor